src/ContentHub.php

```diff
@@ -214,6 +214,23 @@ public function deleteEntity($uuid)
     }
 
     /**
+     * Purges all entities from the Content Hub.
+     *
+     * This method should be used carefully as it deletes all the entities for
+     * the current subscription from the Content Hub.
+     */
+    public function purge() {
+        $list = $this->listEntities();
```

Comment:
ok that is fine. what tool do you use to follow this coding standard? do you have an IDE and a code style checker?